src/login/logind-action.c

```diff
@@ -346,6 +384,17 @@ int manager_handle_action(
                 return 1;
         }
 
+        if (handle == HANDLE_SECURE_ATTENTION_KEY) {
+                if (!is_edge)
+                        return 0;
+
+                if (!action_seat)
+                        return 0;
+
+                log_info("Requesting greeter on seat %s...", action_seat);
+                return manager_handle_action_secure_attention_key (m, is_edge, action_seat);
```

Comment:
as per coding style, no space between function name and `(`

Reply:
Sorry, bad instinct